src/java/org/apache/cassandra/config/Config.java

```diff
@@ -171,6 +171,20 @@ public static Set<String> splitCommaDelimited(String src)
 
     public volatile DurationSpec.LongMillisecondsBound stream_transfer_task_timeout = new DurationSpec.LongMillisecondsBound("12h");
 
+    public volatile DurationSpec.LongMillisecondsBound cms_await_timeout = new DurationSpec.LongMillisecondsBound("120000ms");
```

Comment:
Out of curiosity, how were default values for all new parameters chosen? Methodology?